Marlin/Configuration_adv.h

```diff
@@ -3111,7 +3204,7 @@
     #define Z4_SLEW_RATE                 1
   #endif
 
-  #if AXIS_DRIVER_TYPE_I(L6470)
+  #if AXIS_IS_L64XX(I)
```

Comment:
Are we confident that these settings apply to `L6470`, `L6474`, `L6480`, and `POWERSTEP01`?

Reply:
I cannot test that. but configuration of axes I, J, K... should work the same way it does for X, Y, Z.  In PR 16452 (Improved STMicro L64XX stepper driver support), `#if AXIS_DRIVER_TYPE_X(L6470)` was changed to `#if AXIS_IS_L64XX(X)` in configuration_adv.h, so I did the same for axes I, J, K....